rir/src/runtime/Code.h

```diff
@@ -9,25 +10,22 @@
 
 namespace rir {
 
+typedef SEXP FunctionSEXP;
+typedef SEXP CodeSEXP;
+
 struct Function;
 struct FunctionSignature;
 
-// Code magic constant is intended to trick the GC into believing that it is
-// dealing with already marked SEXP.
-// Note: gcgen needs to be 1, otherwise the write barrier will trigger and
-//       named count is 3 (ie. NAMEDMAX) to make it stable
-//  It also has an unique bitpattern for gp (0xeeee) so that we can keep it
-//  apart from functions
-#define CODE_MAGIC (uint64_t)0x311eeee1a
+#define CODE_MAGIC 0x3111eeee
```

Comment:
can we change that, so it does *not* look like a sexp anymore?

Reply:
Sure, I just punted on choosing the constant. But then I found [this](https://nedbatchelder.com/text/hexwords.html), so I think I'll make it `11c0de11`. And while I'm being ~clever~ obnoxious, maybe I'll change FUNCTION_MAGIC to `ca11ab1e` and DISPATCH_MAGIC to `d7ab1e00`.